src/views/Peers.tsx

```diff
@@ -154,99 +181,141 @@ export const Peers = () => {
+                                     md={5}
+                                     lg={5}
+                                     xl={5}
+                                     xxl={5} span={5}>
+                                    <Row justify="end">
+                                        <Col>
+                                            <Link to="/add-peer" className="ant-btn ant-btn-primary ant-btn-block">Add Peer</Link>
+                                        </Col>
+                                    </Row>
+                                </Col>
+                            </Row>
+                            {failed &&
+                                <Alert message={failed.code} description={failed.message} type="error" showIcon closable/>
+                            }
+                            {loading && <Loading/>}
+                            <Card bodyStyle={{padding: 0}}>
+                                <Table
+                                    pagination={{pageSize, showSizeChanger: false, showTotal: ((total, range) => `Showing ${range[0]} to ${range[1]} of ${total} peers`)}}
+                                    className="card-table"
+                                    showSorterTooltip={false}
+                                    scroll={{x: true}}
+                                    dataSource={dataTable}>
+                                    <Column title="Name" dataIndex="Name"
+                                            onFilter={(value: string | number | boolean, record) => (record as any).Name.includes(value)}
+                                            sorter={(a, b) => ((a as any).Name.localeCompare((b as any).Name))} />
+                                    <Column title="IP" dataIndex="IP"
+                                            sorter={(a, b) => {
+                                                const _a = (a as any).IP.split('.')
+                                                const _b = (b as any).IP.split('.')
+                                                const a_s = _a.map((i:any) => i.padStart(3, '0')).join()
+                                                const b_s = _b.map((i:any) => i.padStart(3, '0')).join()
+                                                return a_s.localeCompare(b_s)
+                                            }}
+                                            render={(text, record, index) => {
+                                                return <ButtonCopyMessage keyMessage={(record as PeerDataTable).key} text={text} messageText={'IP copied!'} styleNotification={{}}/>
+                                            }}
+                                    />
+                                    <Column title="Status" dataIndex="Connected"
+                                            render={(text, record, index) => {
+                                                return text ? <Tag color="green">online</Tag> : <Tag color="red">offline</Tag>
```

Comment:
Codacy has a fix for the issue: [Replace `············································return·text·?·<Tag·color="green">online</Tag>·:·` with `····················return·text·?·(⏎······················<Tag·color="green">online</Tag>⏎····················)·:·(⏎······················`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
                    return text ? (
```